src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -131,6 +131,10 @@
 @SuppressWarnings("FutureReturnValueIgnored")
 public abstract class ExecutionStrategy {
 
+    @Internal
+    public static final String MAX_RESULT_NODES = "__MAX_RESULT_NODES";
+    @Internal
+    public static final String MAX_RESULT_NODES_BREACHED = "__MAX_RESULT_NODES_BREACHED";
```

Comment:
These are @Internal but how would a consumer use them per request ?

Reply:
internal is wrong

Reply:
maybe move it to another class holding general node infos